ddl/ddl_worker.go

```diff
@@ -32,8 +33,12 @@ import (
 	"golang.org/x/net/context"
 )
 
-// RunWorker indicates if this TiDB server starts DDL worker and can run DDL job.
-var RunWorker = true
+var (
+	// RunWorker indicates if this TiDB server starts DDL worker and can run DDL job.
+	RunWorker = true
+	// ddlWorkerID is used for generating the next DDL worker ID.
+	ddlWorkerID = int32(0)
```

Comment:
How about use int64?

Reply:
why? The number of workers is two now,  and there will be no more than ten in the future.

Reply:
The origin type of id is "int", which is 64 bit actually. When i review the code, i was wondering why changed to int32.

Reply:
@ngaut 
Because I want to use "atomic".

Reply:
@coocood 
I use "atomic" to generate the DDL worker ID. And it's only used in log and metric. I think it's OK.